MosaicResidentInformationApi/V1/Gateways/MosaicGateway.cs

```diff
@@ -31,7 +31,7 @@ public MosaicGateway(MosaicContext mosaicContext)
 
             var peopleIds = queryByAddress
                 ? PeopleIdsForAddressQuery(cursor, limit, firstname, lastname, postcode, address)
```

Comment:
This query should also be able to filter the results by context flag

Reply:
I've added the functionality so this now filters by context flag